data-raw/watershed/clear/clear_creek.rmd

```diff
@@ -1,36 +1,28 @@
-[FWS 2013](https://www.fws.gov/lodi/instream-flow/Documents/Clear%20Creek%20Lower%20Rearing%20Final%20Report-1.pdf) (pgs 150-160)    
 
-
-Instream spawning and rearing habitat for Fall Run Chinook Salmon, Spring Run Chinook Salmon, and Steelhead / Rainbow Trout in Clear Creek is based on data from four U.S. Fish and Wildlife Service (USFWS) instream flow evaluations conducted between 2007 and 2013 for the Central Valley Project Improvement Act (CVPIA). These evaluations developed relationships between flow and suitable spawning and rearing habitat area for Clear Creek for "upper" (Whiskeytown Dam to Clear Creek Road) and "lower" (Clear Creek Road to the Sacramento River) Clear Creek. The original data can be found on pages 113 - 117 of FWS 2007 (Upper Clear Creek spawning), pages 144 - 157 of FWS Rearing 2011 (Upper Clear Creek rearing, pages 107 - 109 of FWS Spawning 2011 (Lower Clear Creek spawning), and pages 150 - 160 of FWS 2013 (Lower Clear Creek rearing). [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service combined the habitat data from these four studies and provided them for the entire Clear Creek watershed in a spreadsheet (see ‘~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx’). Mark Gard instructed us to use A3:G26, A30:G53, and A58:G81 of the 'Clear' tabwithin his 'IFIMWUA.xlsx'. These cells represent the WUA in sqft for the lower alluvial (8.81 mi), canyon (7.33 mi), and upper alluvial (2.27 mi) segments. Fall run are only in the lower alluvial segment.
+Instream spawning and rearing habitat for Fall Run Chinook Salmon, Spring Run Chinook Salmon, and Steelhead in Clear Creek are based on data from four U.S. Fish and Wildlife Service (USFWS) instream flow evaluations conducted between 2007 and 2013 for the Central Valley Project Improvement Act (CVPIA). These evaluations developed relationships between flow and suitable spawning and rearing habitat area for Clear Creek for "upper" (Whiskeytown Dam to Clear Creek Road) and "lower" (Clear Creek Road to the Sacramento River) Clear Creek. The original data can be found on pages 113 - 117 of FWS 2007 (Upper Clear Creek spawning), pages 144 - 157 of FWS Rearing 2011 (Upper Clear Creek rearing, pages 107 - 109 of FWS Spawning 2011 (Lower Clear Creek spawning), and pages 150 - 160 of FWS 2013 (Lower Clear Creek rearing). [Mark Gard](mailto:mark_gard@fws.gov) from the U.S. Fish and Wildlife Service combined the habitat data from these four studies and provided them for the entire Clear Creek watershed in a spreadsheet (see ‘~/cvpiaHabitat/data-raw/mark_gard_data/IFIMWUA.xlsx’). Mark Gard instructed us to use A3:G26, A30:G53, and A58:G81 of the 'Clear' tab within his 'IFIMWUA.xlsx'. These cells represent the WUA in sqft for the lower alluvial (8.81 mi), canyon (7.33 mi), and upper alluvial (2.27 mi) segments. Fall run are only in the lower alluvial segment.
```

Comment:
Will you update the text to include "Fall Run spawning is an estimate provided by Clear Creek watershed experts after receiving feedback that the FWS estimates were too low." and remove any reference that doesn't make sense given this

Reply:
fixed